requirements.txt

```diff
@@ -1 +1,2 @@
 numpy>=1.17.4
+mutf8~=1.0
```

Comment:
It seems like mutf8 isn't imported/used anywhere, is this just an artifact before you went with the current implementation?

Reply:
That library exposes functions to encode and decode the modified utf-8 format.
It isn't directly used by the nbt library but the functions can be given to the library as encoder methods.
It made sense to include it here so dependents do not need to specify it.